cmd/registry-replacer/main.go

```diff
@@ -143,7 +143,11 @@ func replacer(
 		}
 
 		getter := githubFileGetterFactory(info.Org, info.Repo, info.Branch)
-		allSourceImages := sets.String{}
+		allReplacementCandidates := sets.String{}
+
+		// We have to skip pruning if we only get empty dockerfiles because it might mean
+		// that we do not have the appropriate permissions.
+		var hasNonEmptyDockerfile bool
```

Comment:
Let's make this discoverable since it will mean that we still have non-conformant configs

Reply:
Done